S2I tests are failing

```bash
$ ./run_tests_with_s2i.sh

[INFO] Deleting: Service name:rest-http. Deleted:false
[INFO] Deleting: BuildConfig name:rest-http. Deleted:false
[INFO] Deleting: DeploymentConfig name:rest-http. Deleted:false
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 2.57 s <<< FAILURE! - in dev.snowdrop.example.UnmanagedOpenShiftIT
[ERROR] testGreetingEndpointWithNameParameter  Time elapsed: 0.046 s  <<< ERROR!
java.lang.NullPointerException
	at dev.snowdrop.example.UnmanagedOpenShiftIT.setup(UnmanagedOpenShiftIT.java:46)

[ERROR] testGreetingEndpoint  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NullPointerException
	at dev.snowdrop.example.UnmanagedOpenShiftIT.setup(UnmanagedOpenShiftIT.java:46)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   UnmanagedOpenShiftIT.setup:46 NullPointer
[ERROR]   UnmanagedOpenShiftIT.setup:46 NullPointer
[INFO] 
[ERROR] Tests run: 3, Failures: 0, Errors: 2, Skipped: 1
[INFO] 
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.2:verify (default) @ rest-http ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 22.270 s
[INFO] Finished at: 2021-12-01T08:34:49+01:00
[INFO] Final Memory: 58M/228M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.22.2:verify (default) on project rest-http: There are test failures.
[ERROR] 
[ERROR] Please refer to /z/dev/client/redhat/forks-github/rest-http-example/target/failsafe-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/README.md
+++ b/README.md
@@ -17,6 +17,14 @@ sh run_tests_with_dekorate.sh
 ## Running Tests on OpenShift using S2i from Source:
 
 ```
+oc create -f .openshiftio/application.yaml
+oc new-app --template=rest-http -p SOURCE_REPOSITORY_URL="https://github.com/snowdrop/rest-http-example" -p SOURCE_REPOSITORY_REF=sb-2.4.x
+
+sleep 30 # needed in order to bypass the 'Pending' state
+# wait for the app to stand up
+timeout 300s bash -c 'while [[ $(oc get pod -o json | jq  ".items[] | select(.metadata.name | contains(\"build\"))  | .status  " | jq -rs "sort_by(.startTme) | last | .phase") == "Running" ]]; do sleep 20; done; echo ""'
+
+# launch the tests without deploying the application
 sh run_tests_with_s2i.sh
 ```
 

